src/UI/Menu/Settings.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import { MenuBlock, SettingsBlock, SettingsTitle, SettingsParam, LeftButton, RightButton, Button} from './Elements';
+import iconButton from '../../assets/images/arrow.svg';
+import {Link} from "react-router-dom";
+import { connect } from 'react-redux';
+import { changeSize, changeTheme, changeColor } from '../../store/actions';
+
+class Settings extends Component {
+  constructor(props) {
+    super(props);
+    this.buttonHandler = props.buttonHandler;
+  }
+
+  render() {
+    const {color, nightmode} = this.props;
+    const difficulty = this.props.size === 3 ? "hard" : this.props.size === 4 ? "normal" : "easy";
```

Comment:
Please use an if-statement for long expressions like this one to make it more readable.